docs_user/assemblies/assembly_adopting-the-data-plane.adoc

```diff
@@ -0,0 +1,23 @@
+[id="adopting-data-plane_{context}"]
+
+:context: adopting-data-plane
+
+= Adopting the data plane
+
+Adopting the {rhos_long} data plane involves the following steps:
+
+* Stopping any remaining services on the {rhos_prev_long} ({OpenStackShort}) control plane.
+* Deploying the required custom resources.
+* If applicable, performing a fast-forward upgrade on Compute services from {OpenStackShort} {rhos_prev_ver} to {rhos_acro} {rhos_curr_ver}.
+
+[WARNING]
+This step is a "point of no return" in the data plane adoption
+procedure. The source control plane and data plane services must not
```

Comment:
````
The source control plane and data plane services must not be re-enabled after the data plane is deployed and the control plane has taken control of the data plane.
````
Make this sentence active. Perhaps like this?
````
After the RHOSO control plane is managing the newly deployed data plane, you must not re-enable services on the RHOSP 17.1 control plane and data plane.
````
Thought: should we add a consequence for re-enabling services on the RHOSP 17.1 data and control planes?
Like?

````
Re-enabling the services causes ______ <something bad> data corruption? hangs the system? 
````

Reply:
@bogdando What are the consequences of re-enabling the data plane services?

Reply:
That would be a classic "split brain" for workloads wither being managed by a two control planes, or a two data plane services, resulting in data corruption, or any other undesired and unrecoverable outcomes, like fully loosing of control over existing workloads and inability to start new ones.

@jistr does this description look OK to you?

Reply:
I  like your description. I'd tweak it a little bit to this: "Re-enabling services causes workloads to be managed by two control planes or two data planes, resulting in data corruption, losing control of existing workloads, inability to start new workloads, or other issues."